src/dagmc.cpp

```diff
@@ -70,34 +160,68 @@ void load_dagmc_geometry()
     }
 
     if (model::DAG->is_implicit_complement(vol_handle)) {
-      // assuming implicit complement is void for now
+      // assuming implicit complement is always void
```

Comment:
IIRC DMD will allow the implicit complement to be set to a given material, you just need to add any volume to the group "mat:Steel_comp" should allow the implicit complement to be set.

Reply:
Right you are! I'll add support for that.